docs/next.config.js

```diff
@@ -165,6 +165,14 @@ module.exports = withNextPluginPreval({
         destination: '/react/components/accordion',
         permanent: false,
       },
+      /**
+       * Renamed "Storage Manager" to "File Uploader"
+       */
+      {
+        source: '/react/connected-components/storage/storagemanager',
+        destination: '/react/connected-components/storage/fileuploader',
+        permanent: true,
+      },
```

Comment:
Will this cover redirects from the primary docs that are linking to the existing StorageManager documentation?

Reply:
yes, it should

Reply:
I believe so. This is the only endpoint linked in the primary docs. There is also a PR under review which updates the links to use the `/fileuploader` endpoint instead: https://github.com/aws-amplify/docs/pull/7921